### PR TITLE
arm64-kubernetes-kubesphere.md

### PR DESCRIPTION
**Pull request by faweizhao26**
Signed-off-by: faweizhao26 <faweizhao26@gmail.com>

### CODE REVIEW

File: content/zh/blogs/arm64-kubernetes-kubesphere.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+#### 3. 执行安装
+```
+# 以上准备工作完成且再次检查配置文件无误后，执行安装。
+./kk create cluster -f config-sample.yaml --with-kubesphere
+```
+
+### 查看结果
+
+![](https://pek3b.qingstor.com/kubesphere-community/images/查看结果-1.png)
```
Comment (FeynmanZhou) [inline content/zh/blogs/arm64-kubernetes-kubesphere.md:113]: Please rename these images name in English.

Comment (faweizhao26) [reply content/zh/blogs/arm64-kubernetes-kubesphere.md]: I have modified it.

File: content/zh/blogs/arm64-kubernetes-kubesphere.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
+---
+title: '三步搞定 ARM64 离线部署 Kubernetess + KubeSphere'
```
Comment (FeynmanZhou) [inline content/zh/blogs/arm64-kubernetes-kubesphere.md:2]: Kubernetess => Kubernetes

File: content/zh/blogs/arm64-kubernetes-kubesphere.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+### 背景
+
+随着我国对信息安全的愈发重视，国产化的趋势也越来越浓，包括国产操作系统、国产 CPU 等。由于 ARM 架构国产 CPU 在维持创新可信和先进性方面的潜在优势，其应用也将会越来越广泛。
+
+KubeSphere 作为一款深受国内外开发者所喜爱的开源容器平台，也将积极参与并探索在 ARM 架构下的应用与创新。本文将主要介绍如何在 ARM64 环境下部署 Kubernetes 和 KubeSphere。
+
+### 环境准备
+#### 节点
+kubeSphere 支持的操作系统包括：
```
Comment (FeynmanZhou) [inline content/zh/blogs/arm64-kubernetes-kubesphere.md:17]: kubeSphere => KubeSphere

**Review by FeynmanZhou — CHANGES_REQUESTED** [review]
See my comments above.

**Comment by faweizhao26** [comment]
> See my comments above.

OK，I have modified these typos.

**Review by FeynmanZhou — APPROVED** [review]

**Pull request merged**